cockatrice/src/settings/gamefilterssettings.cpp

```diff
@@ -92,6 +92,17 @@ int GameFiltersSettings::getMaxPlayers()
     return previous == QVariant() ? 99 : previous.toInt();
 }
 
+void GameFiltersSettings::setMaxGameAgeAsInt(int maxGameAgeEnum)
+{
+    setValue(maxGameAgeEnum, "max_game_age_enum", "filter_games");
+}
+
+int GameFiltersSettings::getMaxGameAgeEnum()
+{
+    QVariant previous = getValue("max_game_age_enum", "filter_games");
+    return previous == QVariant() ? 5 : previous.toInt();
```

Comment:
Why are we defaulting to 5 on this (which means to filter 2+ hours)? This could easily get scrambled if values change order or smth. Can we default this to MaxGameAge::SOMETHING that way it's clear what this value is meant to be?

Reply:
Clearing filters means to remove any filters.
If the age filter stays applied at "2+ h" and certain games stay hidden by default that might not be what the button suggests and the user expects.

Reply:
Done. Decided to store the seconds itself rather than an enum here, and propagated changes throughout (pushing branch momentarily).